scripts/README_STABLE_BUILD.md

```diff
@@ -0,0 +1,35 @@
+<h1 align="center"><img src="./logo_white.png" alt="Logo" height="128" /></h1>
+<h2 align="center">Prisma VS Code Extension</h2>
+<div align="center">
+
+[![Version](https://vsmarketplacebadge.apphb.com/version/prisma.Prisma.svg)](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma)
+[![Installs](https://vsmarketplacebadge.apphb.com/installs/prisma.Prisma.svg)](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma)
+[![Ratings](https://vsmarketplacebadge.apphb.com/rating/prisma.Prisma.svg)](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma)
+
+</div>
+<hr>
+Adds syntax highlighting, formatting, jump-to-definition and linting for `.prisma` files.
+
+This plugin is designed for [Prisma 2](https://www.prisma.io/blog/announcing-prisma-2-zq1s745db8i5). Information about the new datamodel syntax can be found [here](https://github.com/prisma/prisma2/blob/master/docs/data-modeling.md).
+
+## Features
+
+* Syntax highlighting
+* Auto-formatting
+* Linting
+* Jump-to-definition
+* Auto-completion
+
+It also includes an End-to-End test.
+
+## Install
+
+* Get the Prisma Extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma).
+
+## Preview
+
+![Preview Schema](https://i.imgur.com/W80iRwE.png)
+
+## Contributing
+
+Read more about [how to contribute to the Prisma VSCode extension](CONTRIBUTING.md)
```

Comment:
Will this link even work on VSCode Marketplace?

Reply:
yes, this works on the Marketplace